### DISCUSSION TITLE
Ensuring safety.

### DISCUSSION THREAD

**Discussion by michail935** [deforum-art/sd-webui-text2video#91] [General]
Hello to the developers. I don't know where or to whom to send these messages. But yesterday while checking Defender I found this problem:
Trojan:WIN32/Sirefef!cfg
containerfile :\\stable-diffusion-webui\models\ModelScope\t2v\VQGAN_autoencoder.pth
файл :\stable-diffusion-webui\models\ModelScope\t2v\VQGAN_autoencoder.pth→архив /data/ 900
This is the second time I have encountered such a problem in programs related to stable diffusion. Perhaps it is a mistake, but it is better to let you know, so that there were no problems. If that's really the case, it's not good that someone is doing this in programs on GitHub.

**Comment by hithereai** [comment]
https://www.virustotal.com/gui/url/4de38c8d24d03b6ca234c7dd2d275f0be265175c3d7dc6f02739875d9b65d8eb?nocache=1

https://www.virustotal.com/gui/url/5b939a96cd15f44e1c9de2e781e6702b845a6c5a20b5fbf007e48f4de7f93702?nocache=1

**Reply by michail935** [reply]
Hello.  Is it possible that Defender is giving out errors? If you know, let me know the answer.
